tutorial/creating_packages/configure_options_settings.rst

```diff
@@ -0,0 +1,269 @@
+explain here, recipes themselves can even adjust their package ID calculations, we can
+have different recipe and package revisions besides package IDs and there's also a
+built-in mechanism in Conan that can be configured to declare that some packages with a
+certain package ID are compatible with other.
+
+Maybe you have now the intuition of why we delete settings or options in Conan recipes.
+If you do that, those values will not be added to the computation of the package ID, so
+even if you define them, the resulting package ID will be the same. You can check this
+behaviour, for example with the fPIC option that is deleted when we build with the
+option ``shared=True``. Regardless of the value you pass for the fPIC option the generated
+package ID will be the same for the **hello/1.0** binary:
+
+.. code-block:: bash
+    
+    $ conan conan create . --build=missing -s compiler.cppstd=gnu11 -o shared=True -o fPIC=True -tf=None
+    ...
+    hello/1.0 package(): Packaged 1 '.h' file: hello.h
+    hello/1.0 package(): Packaged 1 '.dylib' file: libhello.dylib
+    hello/1.0: Package '2a899fd0da3125064bf9328b8db681cd82899d56' created
+    hello/1.0: Created package revision f0d1385f4f90ae465341c15740552d7e
+    hello/1.0: Full package reference: hello/1.0#e6b11fb0cb64e3777f8d62f4543cd6b3:2a899fd0da3125064bf9328b8db681cd82899d56#f0d1385f4f90ae465341c15740552d7e
+    hello/1.0: Package folder /Users/carlosz/.conan2/p/8a55286c6595f662/p
+
+    $ conan conan create . --build=missing -s compiler.cppstd=gnu11 -o shared=True -o fPIC=False -tf=None
+    ...
+    -------- Computing dependency graph --------
+    Graph root
+        virtual
+    Requirements
+        fmt/8.1.1#601209640bd378c906638a8de90070f7 - Cache
+        hello/1.0#e6b11fb0cb64e3777f8d62f4543cd6b3 - Cache
+
+    -------- Computing necessary packages --------
+    Requirements
+        fmt/8.1.1#601209640bd378c906638a8de90070f7:d1b3f3666400710fec06446a697f9eeddd1235aa#24a2edf207deeed4151bd87bca4af51c - Skip
+        hello/1.0#e6b11fb0cb64e3777f8d62f4543cd6b3:2a899fd0da3125064bf9328b8db681cd82899d56#f0d1385f4f90ae465341c15740552d7e - Cache
+
+    -------- Installing packages --------
+
+    -------- Installing (downloading, building) binaries... --------
+    hello/1.0: Already installed!
+
+As you can see, the first run created the ``2a899fd0da3125064bf9328b8db681cd82899d56``
+package, and the second one, regardless of the different value of the fPIC option, said we
+already had the ``2a899fd0da3125064bf9328b8db681cd82899d56`` package installed.
+
+C libraries
+^^^^^^^^^^^
+
+There are other typical cases where you want to delete certain settings. Imagine that you
+are packaging a C library. When you build this library, there are settings like the
+compiler C++ standard (``settings.compiler.cppstd``) or the standard library used
+(``self.settings.compiler.libcxx``) that won't affect the resulting binary at all. Then it
```

Comment:
This goes beyond the ``package_id``. When used in ``configure()`` you are also removing them from all the toolchain, and the build system integrations will do a different thing, because the C++ settings do not exist. 

Finally, it will not be part of the ``package_id``, but the main different with respect to being removed in ``package_id()`` is that they are discarded from the build process too

Reply:
I can add a note. I did not want to give too many details so that users start to digest the concepts little by little...